lib/models/BooksModel.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+
+class BooksModel {
+  final String type; // type will bw book !!
+  final String id;
+  final double price;
+  final bool isDealOfTheDay;
```

Comment:
boolean is not required , instead add an argument of type integer which defines the quantity. for example - int quantity